Library/Homebrew/software_spec.rb

```diff
@@ -46,6 +47,7 @@ def initialize
     @build = BuildOptions.new(Options.create(@flags), options)
     @compiler_failures = []
     @bottle_disable_reason = nil
+    @macos_deps = []
```

Comment:
Feels a bit weird having a `macos` variable/method that's declared here rather than in `extend/os/Mac/software_spec.rb`. What about something named more like `uses_from_macos` which describes the DSL?

Reply:
How might I create a new instance variable in `extend/os/mac/software_spec.rb` without retyping the contents of `def initialize` from `software_spec.rb`?

Reply:
Got it working in the extend file instead.